kartothek/io/dask/common_cube.py

```diff
@@ -101,7 +101,7 @@ def build_cube_from_bag_internal(
 
     Parameters
     ----------
-    data:
+    data: dask.bag.Bag
```

Comment:
Why did you add the type here? Shouldn't it just be `data` (without colon) to be consistent with other changes (and numpy docstring style)?

Reply:
yes, indeed. Sometimes my "regex search and replace" hammer was a bit too forceful, I assume. In particular with the cube API I had issues since the API is using a lot of type annotations in docs w/out having proper type annotations in general. Sometimes I fixed that, sometimes I didn't. TBH, I consider this PR one of the "it's not 100% but it still an improvement" things :)

If it is just about this line in particular, I can address this but I am affraid this or similar mistakes were made and I would prefer not to go over the code base again. This can be _eventually_ cleaned up since having the type documented twice should generally not be harmful

Reply:
Alright no problem, I just wanted to know if this was an intentional change.